server/sonar-web/src/main/webapp/WEB-INF/app/models/event.rb

```diff
@@ -63,6 +63,6 @@ def self.already_exists(snapshot_id, event_name, event_category)
   # TODO: Remove this code when everything has been checked on the Event handling, both on the UI and the WS API
   #
   def populate_snapshot
-    self.snapshot=Snapshot.snapshot_by_date(resource_id, event_date) unless self.snapshot
+    self.snapshot=Snapshot.snapshot_by_date(resource_id, event_date.to_i*1000) unless self.snapshot
```

Comment:
Method snapshot_by_date still requires a DateTime, so this change must be reverted.